examples/vae_simple.py

```diff
@@ -0,0 +1,322 @@
+import sys
+import os
+import numpy as np
+import theano
+import theano.tensor as T
+import lasagne as nn
+import time
+import pickle
+from PIL import Image
+from lasagne.layers import MergeLayer
```

Comment:
I'd remove that import and write `class GaussianSampleLayer(nn.layers.MergeLayer)` below, but that's mainly cosmetic.